src/xrGame/player_hud.cpp

```diff
@@ -547,36 +547,33 @@ u32 player_hud::motion_length(const MotionID& M, const CMotionDef*& md, float sp
     }
     return 0;
 }
-const Fvector& player_hud::attach_rot() const
-{
-    if (m_attached_items[0])
-        return m_attached_items[0]->hands_attach_rot();
-    else if (m_attached_items[1])
-        return m_attached_items[1]->hands_attach_rot();
-    else
-        return Fvector().set(0, 0, 0);
-}
-
-const Fvector& player_hud::attach_pos() const
```

Comment:
I suggest to leave this methods. Consider changing the prototype to void player_hud::attach_pos(Fvector& out) const